src/marqo/tensor_search/models/search.py

```diff
@@ -0,0 +1,42 @@
+import json
+from pydantic import BaseModel
+from typing import Any, Union, List, Dict, Optional, NewType
+
+Qidx = NewType('Qidx', int)
+JHash = NewType('JHash', int)
```

Comment:
Maybe add comments/ expand these names so that they make sense to a dev with less context?

`QueryIndex = ...  # Indicates the position of a search query in a bulk search request`
`VectoriseJobHash = ....  # hash of a VectoriseJob`